sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statements.scala

```diff
@@ -343,6 +343,15 @@ case class ShowTableStatement(
     partitionSpec: Option[TablePartitionSpec])
   extends ParsedStatement
 
+/**
+ * A SHOW VIEWS statement, as parsed from SQL.
+ */
+case class ShowViewsStatement(
+    namespace: LogicalPlan,
+    pattern: Option[String]) extends ParsedStatement {
+  override def children: Seq[LogicalPlan] = Seq(namespace)
+}
```

Comment:
It seems we only provide views in v1 SessionCatalog, so I implemented ShowViews along with v1 style.